test/functional/test_framework/test_framework.py

```diff
@@ -477,10 +477,8 @@ def _initialize_chain(self):
             for node in self.nodes:
                 node.wait_for_rpc_connection()
 
-            # For backward compatibility of the python scripts with previous
-            # versions of the cache, set mocktime to Jan 1,
-            # 2014 + (201 * 10 * 60)"""
-            self.mocktime = 1388534400 + (201 * 10 * 60)
+            # Create the blocks, spaced 10 minutes apart, starting now
+            self.mocktime = int(time.time())
```

Comment:
Seems arbitrary to still use mocktime, but only change the magic value. Also, wouldn't mining blocks with timestamps in the future lead to block sync issues? IIRC a node woulnd't even attempt to download a block when the header has a future timestamp?

I'd prefer to remove mocktime altogether and only use it in tests that require it for some reason (see the test below you modified for example)

Just my opinion, no need to fix in this pull request.